.github/workflows/tests.yaml

```diff
@@ -0,0 +1,116 @@
+          - ci/travis/36-pd025.yaml
+          - ci/travis/37-latest-defaults.yaml
+          - ci/travis/37-latest-conda-forge.yaml
+          - ci/travis/38-latest-conda-forge.yaml
+        include:
+          - env: ci/travis/37-latest-conda-forge.yaml
+            os: macos-latest
+            postgis: false
+            dev: false
+          - env: ci/travis/38-latest-conda-forge.yaml
+            os: macos-latest
+            postgis: false
+            dev: false
+          - env: ci/travis/37-latest-conda-forge.yaml
+            os: windows-latest
+            postgis: false
+            dev: false
+          - env: ci/travis/38-latest-conda-forge.yaml
+            os: windows-latest
+            postgis: false
+            dev: false
+          - env: ci/travis/38-latest-conda-forge.yaml
+            os: ubuntu-latest
+            postgis: true
+            dev: false
+          - env: ci/travis/37-dev.yaml
+            os: ubuntu-latest
+            postgis: false
+            dev: true
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Conda Env
+        uses: goanpeca/setup-miniconda@v1
+        with:
+          auto-update-conda: true
+          auto-activate-base: false
+          activate-environment: test
+          environment-file: ${{ matrix.env }}
+          channel-priority: strict
+
+      - name: Check and Log Environment
+        shell: bash -l {0}
+        run: |
+          python -V
+          conda info
+          conda list 2>&1 | tee conda.txt
+          if ( cat conda.txt | grep -q pygeos  )
+          then
+            echo "Setting HAS_PYGEOS=1"
+            echo '::set-env name=HAS_PYGEOS::1'
+          else
+            echo "Setting HAS_PYGEOS=0"
+            echo '::set-env name=HAS_PYGEOS::0'
+          fi
+
+      - name: Test without PyGEOS
+        shell: bash -l {0}
+        continue-on-error: ${{ matrix.dev }}
+        env:
+          USE_PYGEOS: 0
+        run: |
+          pytest -v -r s -n auto --color=yes --cov=geopandas --cov-append --cov-report term-missing --cov-report xml geopandas/
+
+      - name: Test with PyGEOS
+        shell: bash -l {0}
+        continue-on-error: ${{ matrix.dev }}
+        if: env.HAS_PYGEOS == 1
+        env:
+          USE_PYGEOS: 1
+        run: |
+          pytest -v -r s -n auto --color=yes --cov=geopandas --cov-append --cov-report term-missing --cov-report xml geopandas/
+
+      - name: Test with PostGIS
+        shell: bash -l {0}
+        if: matrix.postgis == true
+        env:
+          PGUSER: postgres
+          PGPASSWORD: postgres
+          PGHOST: "127.0.0.1"
+        run: |
+          conda install postgis -c conda-forge
+          source ci/travis/setup_postgres.sh
+          pytest -v -r s -color=yes --cov=geopandas --cov-append --cov-report term-missing --cov-report xml geopandas/io/tests/test_sql.py | tee /dev/stderr | if grep SKIPPED >/dev/null;then echo "TESTS SKIPPED, FAILING" && exit 1;fi
```

Comment:
Alternatively, you could also do this check after the normal tests? (instead of splitting out the postgis tests?)

Reply:
Done